src/storage-client/src/controller.rs

```diff
@@ -1772,14 +1772,14 @@ where
 
     fn drop_sinks_unvalidated(&mut self, identifiers: Vec<GlobalId>) {
         for id in identifiers {
-            let export = match self.export(id) {
-                Ok(export) => export,
-                Err(_) => continue,
-            };
+            // Already removed.
+            if self.export(id).is_err() {
+                continue;
+            }
 
-            let read_capability = export.read_capability.clone();
-            let storage_dependencies = export.storage_dependencies.clone();
-            self.remove_read_capabilities(read_capability, &storage_dependencies);
+            // We don't explicitly call `remove_read_capabilities`! Downgrading the
+            // frontier of the source to `[]` (the empty Antichain), will propagate
```

Comment:
did you mean to say "source" or "sink" here?

Reply:
meant sink